### ISSUE TITLE
Annotations from Noctua models not reaching Xenbase GPAD on snapshot.

### ISSUE THREAD

**Issue by malcolmfisher103** [geneontology/noctua-models#146]
In the time between June 14th and June 22nd nearly 300 annotations have disappeared from the Xenbase GPAD file (http://snapshot.geneontology.org/products/annotations/noctua_xenbase.gpad.gz). The missing annotations can still be found in the latest release version (http://release.geneontology.org/2020-06-01/products/annotations/noctua_xenbase.gpad.gz). Please find attached a file with the annotations that have been lost ([Missing.lines.txt](https://github.com/geneontology/noctua-models/files/4838529/Missing.lines.txt)).

 Some of these may be the result of legitimate changes in models but in many cases entire models are no longer represented. I can still go to the models in Noctua and see the annotations in the 'Workbenches' menu's 'Annotation preview' or 'Export GPAD' option from the 'Model' menu, see [this model](http://noctua.berkeleybop.org/editor/graph/gomodel:581e072c00000295?model_id=gomodel:581e072c00000295) for an example.

Have any extra  QC checks been added in the last couple of weeks that might account for this? I am not seeing any obvious problem or commonalities with the models that have been lost.

**Comment by kltm** [comment]
@malcolmfisher103 To confirm: the models are still in Noctua (i.e. http://noctua.geneontology.org/editor/graph/gomodel:581e072c00000295 (noting the corrected domain)), but you are not seeing them in the output GPAD?

If so, and the models are making it to GitHub, this will need input from @ukemi and @balhoff , and possibly get moved to the minerva tracker.

**Comment by malcolmfisher103** [comment]
@kltm Yes, the models are in Noctua and in the [turtle file](https://github.com/geneontology/noctua-models/blob/master/models/581e072c00000295.ttl) on GitHub but not in the output GPAD.

**Comment by ukemi** [comment]
@kltm, this is correct. The models are there, but in the recent past something about the gpad output has changed and there are now missing annotations. The particular model above is a mixed species model, which may explain the problem. But, I don't think the one was that we looked at together this morning was it  @malcolmfisher103?

**Comment by ukemi** [comment]
I just split out the human process from the frog process (GO:0001558).

**Comment by malcolmfisher103** [comment]
There are other purely frog examples such as '[gomodel:5a7e68a100001201](http://noctua.geneontology.org/editor/graph/gomodel:5a7e68a100001201?model_id=gomodel:5a7e68a100001201)', the one we looked at this morning, which involves several genes whose knockdown disrupted swimming.

**Comment by ukemi** [comment]
@kltm, this seems to be a GPAD output issue. Maybe we should move this to the Minerva tracker.

**Comment by malcolmfisher103** [comment]
Maybe that does bring in a commonality since both of these are cases where distinct molecular functions enabled by distinct gene products are linked to the same GO biological process entity.

**Comment by ukemi** [comment]
That should be allowed. Eventually we want to be able to causally link up all those functions.
